apps/benchmark/README.md

```diff
@@ -127,3 +128,16 @@ Build TVM with LLVM and ROCm enabled. [Help](https://tvm.apache.org/docs/install
 ```bash
 python3 gpu_imagenet_bench.py --model gfx900 --target rocm
 ```
+
+### Adreno GPU
+
+Adreno benchmarks are automated over the docker - [ci_adreno](https://github.com/apache/tvm/blob/main/docker/Dockerfile.ci_adreno).
+Adreno docker share the Android devices from host. It is adviced to have host adb version same as docker, which is ```1.0.41```
+
+Below command runs all the benchmarks over given Android device.
+```bash
+export ANDROID_SERIAL=<ADB ID>
+./tests/scripts/ci.py adreno -b
```

Comment:
@srkreddy1238, @driazati Do you have any ideas, why after running this command I got the following error?

```
Running 'bash .ci-py-scripts/ci-adreno-prhyz.sh' inside ci_adreno...
Unable to find image 'ci_adreno:latest' locally
docker: Error response from daemon: pull access denied for ci_adreno, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
``` 

It is not the first time when I don't have access to the `adreno_ci` docker image. I would like to have this access.

Reply:
It's not an access issue, the adreno Docker image was never added to the [tlcpack docker hub](https://hub.docker.com/u/tlcpack) so there is nothing to download. You'll need to build the image locally (`bash docker/build.sh ci_adreno`) or fix it so adreno is uploaded along with the other images.